packages/console/src/hooks/use-ui-languages.ts

```diff
@@ -0,0 +1,39 @@
+import { builtInLanguages as builtInUiLanguages } from '@logto/phrases-ui';
+import { useMemo } from 'react';
+import useSWR from 'swr';
+
+import { CustomPhraseResponse } from '@/types/custom-phrase';
+
+import { RequestError } from './use-api';
+
+const useUiLanguages = () => {
+  const {
+    data: customPhraseList,
+    error,
+    mutate,
+  } = useSWR<CustomPhraseResponse[], RequestError>('/api/custom-phrases');
+
+  const isLoading = useMemo(() => !customPhraseList && !error, [customPhraseList, error]);
```

Comment:
Simple variable that does not require complex computation does not have to use `useMemo`. Just put:
```typescript
const isLoading = !customPhraseList && !error;
```

Reply:
updated